src/System.Data.Common/ref/System.Data.Common.manual.cs

```diff
@@ -15,6 +15,35 @@ internal class ConstraintConverter { }
     [System.ComponentModel.TypeConverter(typeof(RelationshipConverter))]
     public partial class DataRelation { }
     internal class RelationshipConverter { }
+
+    public partial class DataColumn
+    {
+        [System.ComponentModel.DefaultValueAttribute(typeof(string))]
+        [System.ComponentModel.RefreshPropertiesAttribute(System.ComponentModel.RefreshProperties.All)]
+        [System.ComponentModel.TypeConverter(typeof(ColumnTypeConverter))]
+        public System.Type DataType { get { throw null; } set { } }
+
+        [System.ComponentModel.TypeConverter(typeof(DefaultValueTypeConverter))]
+        public object DefaultValue { get { throw null; } set { } }
+    }
+    internal class ColumnTypeConverter { }
+    internal class DefaultValueTypeConverter { }
```

Comment:
Having public attributes referring to internal types doesn't cause problems?

Reply:
Not a problem and something we're already doing, it's necessary from the tooling perspective.  They invoke the internal converters.  If we were to do this now I'd say we should make these public instead.